diagnosis/getkuberneteslogs.sh

```diff
@@ -128,56 +121,96 @@ done
 if [ -z "$USER" ]
 then
     echo ""
-    echo "[ERR] --user is required"
+    echo -e "$(date) [Err] --user is required"
     printUsage
 fi
 
 if [ -z "$IDENTITYFILE" ]
 then
     echo ""
-    echo "[ERR] --identity-file is required"
+    echo -e "$(date) [Err] --identity-file is required"
     printUsage
 fi
 
-if [ -z "$DVM_HOST" -a -z "$MASTER_HOST" ]
+if [ -z "$DVM_HOST" ]
```

Comment:
This can not be mandatory as AKS-Engine will not create DVM when run using commandline.